ext/reflection/reflection.stub.php

```diff
@@ -425,6 +425,11 @@ public function getType() {}
 
     /** @return bool */
     public function hasType() {}
+
+    /** @return bool */
+    public function hasDefaultValue() {}
```

Comment:
As this is a new method, I think we can go for 
```suggestion
    public function hasDefaultValue(): bool {}
```
?

Assuming that we haven't discussed/agreed already if we don't want return types in API for classes' methods.

Reply:
Adding them to new methods is fine, it is not possible to add them to old non final methods because of the BC break it would entail.